lib/cli/cli.js

```diff
@@ -148,10 +148,18 @@ CLI.prototype.callHelp = function(options) {
 };
 
 CLI.prototype.logError = function(error) {
+  var result = 1;
   if (this.testing && error){
-    console.error(error.message);
-    console.error(error.stack);
+    debug(error.message);
```

Comment:
@stefanpenner I put this behind a debug statement because it was noisy and confusing to see the intentional errors from testing failure cases in the test logs, but I talked to @lukemelia about it (who originally added it) and his reasoning was that it was previously obscuring errors during testing. I'm not quite sure how to best hide the errors when we are intentionally tripping them, but surfacing them when we need to, and was wondering if you had any input on how to handle this.